vars/openShift.groovy

```diff
@@ -64,8 +64,9 @@ def withUINode(Map parameters = [:], Closure body) {
         'namespace',
         cloud.equals(pipelineVars.defaultUICloud) ? pipelineVars.defaultUINameSpace : pipelineVars.defaultNameSpace
     )
-    slaveImage = parameters.get('slaveImage', pipelineVars.jenkinsSlaveIqeImage)
+    slaveImage = parameters.get('slaveImage', pipelineVars.centralCIjenkinsSlaveImage)
```

Comment:
Yeah, the main purpose is to reduce the number of images we use. Resource consumption is a valid point. Definitely, jenkins-slave and iqe-tests don't need many resources. So it can be reduced.